.tool-versions

```diff
@@ -1 +1 @@
-nodejs 20.18.1
+nodejs v20.18.1
```

Comment:
This is supposed to have the v?

Reply:
Yeah, I think so; I'm getting warnings w/o it.